tutorials/algorithms/11_VarQTE.ipynb

```diff
@@ -0,0 +1,925 @@
+          },
+          "metadata": {}
+        }
+      ],
+      "source": [
+        "import pylab\n",
+        "\n",
+        "h_exp_val = np.array([ele[0][0] for ele in evolution_result.observables])\n",
+        "\n",
+        "exact_h_exp_val = sol.observables[0][0].real\n",
+        "\n",
+        "times = evolution_result.times\n",
+        "pylab.plot(times, h_exp_val, label= \"VarQITE\")\n",
+        "pylab.plot(times, exact_h_exp_val , label= \"Exact\",  linestyle='--')\n",
+        "pylab.xlabel(\"Time\")\n",
+        "pylab.ylabel(r\"$\\langle H \\rangle$ (energy)\")\n",
+        "pylab.legend(loc=\"upper right\");"
+      ]
+    },
+    {
+      "cell_type": "code",
+      "execution_count": 10,
+      "metadata": {
+        "colab": {
+          "base_uri": "https://localhost:8080/"
+        },
+        "id": "-2uSBEO83vU3",
+        "outputId": "369b4864-f0b0-4096-8a59-4d104bf841b1"
+      },
+      "outputs": [
+        {
+          "output_type": "stream",
+          "name": "stdout",
+          "text": [
+            "Ground state energy -2.0097479079521197\n"
+          ]
+        }
+      ],
+      "source": [
+        "print(\"Ground state energy\", h_exp_val[-1])"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "8dp41GBm3TIS"
+      },
+      "source": [
+        "as the above figure indicated, we have obtained the converged ground state energy."
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "zSGo5QcUlePi"
+      },
+      "source": [
+        "## Efficient classical calculation of gradients with VarQITE\n",
+        "\n",
+        "You can use classically efficient gradient calculations to speed up the time evolution simulation by setting `qgt` to `ReverseQGT()` and `gradient` to `ReverseEstimatorGradient()`."
+      ]
+    },
+    {
+      "cell_type": "code",
+      "execution_count": 11,
+      "metadata": {
+        "id": "6PzxWQyIld-T"
+      },
+      "outputs": [],
+      "source": [
+        "from qiskit.algorithms.gradients import ReverseEstimatorGradient, ReverseQGT\n",
+        "\n",
+        "var_principle = ImaginaryMcLachlanPrinciple(qgt = ReverseQGT() , gradient = ReverseEstimatorGradient())\n",
+        "evolution_problem = TimeEvolutionProblem(hamiltonian, time, aux_operators=aux_ops)\n",
+        "var_qite = VarQITE(ansatz, init_param_values, var_principle, Estimator())\n",
+        "evolution_result_eff = var_qite.evolve(evolution_problem)"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "S8mtNpT5NirQ"
+      },
+      "source": [
+        "It takes only $1$ minute to calculate imaginary time evolution. The execution time is reduced by about 33% (this may vary for each execution, but generally results in a speedup)."
```

Comment:
```suggestion
        "In this example, it takes only $1$ minute to calculate imaginary time evolution. The execution time is reduced by about 33% (this may vary for each execution, but generally results in a speedup)."
```